tests/buildTools/src/test/scala/tests/MillBuildToolSuite.scala

```diff
@@ -0,0 +1,32 @@
+package tests
+
+class MillBuildToolSuite extends BaseBuildToolSuite {
```

Comment:
So this doesn't pass right now because to my understanding the tests rely on being able to pass in `targetRoot` which `mill-scip` doesn't support. I didn't really see a reason to when creating it, but I guess I could add it if it's a must to be used here. Let me know, and I'll hold off until getting feedback.